content/articles/wordpress-penetration-testing/index.md

```diff
@@ -0,0 +1,131 @@
+
+Pentesting provides you with intelligence and insights into how to mature your WordPress security. Simply, you become the hacker to secure your site. This is by understanding how you are likely to be attacked and what steps to take to secure your site.
+
+The penetration testing process identifies the system's vulnerabilities, system exploitation, discovering vulnerabilities, and reporting.
+
+### How hackers gain access to websites
+A hacker put together a lot of information about your site. An attacker's main aim is to get into your website, to get into the WP admin. Once he can access the WP admin, he knows these things:
+- WordPress version
+- Theme
+- Plugins and their versions
+- User enumeration
+
+If the hackers find [user enumeration](https://www.rapid7.com/blog/post/2017/06/15/about-user-enumeration/), then they know your site is vulnerable. This is how hackers gain access to websites:
+- User enumeration. Hackers use brute-force attack techniques to guess or confirm legit users in a system.
+- Using vulnerable plugins and themes.
+- [SQL injections attacks](https://www.section.io/engineering-education/how-to-fix-and-prevent-sql-injection-in-wordpress/).
+- [Cross-site scripting (XSS) attacks](https://www.section.io/engineering-education/how-to-prevent-cross-site-scripting-in-node-js/).
+- [Cross-site request forgery (CSRF) attacks](https://www.section.io/engineering-education/understanding-csrf/).
+- [Social engineering attacks](https://www.imperva.com/learn/application-security/social-engineering-attack/).
+
+To avoid falling into the trap of being hacked, make sure:
+- Your WordPress, plugins, and themes are updated regularly.
+- WP-Hardening for L1 security. Ensure you install the [WP hardening](https://wordpress.org/plugins/wp-security-hardening/) plugin for WP security.
+- Scan your server regularly. Be security conscious.
+
+### Penetration testing methodology
+A penetration testing methodology is a systematic approach used to identify vulnerabilities and weaknesses in the IT infrastructure. This systematic approach gives you the utmost amount of information about the security outlook of your system, network, website, or application.
+
+While doing penetration testing, break it down into the following steps:
+1. **Reconnaissance**. In this phase, you gather information about your system, network, and servers.
+2. **Scanning**. In this phase, you run a scan on your site to find vulnerabilities.
+3. **Exploitation**. In this phase, you test any possible exploitation of flaws identified in the previous phase. The exploitation of security vulnerabilities allows assessing their impact.
+4. **Mitigation**. This phase removes the vulnerabilities you have found from your system, network, and servers.
+
+### Using kali Linux on VirtualBox for WordPress security
+Getting started with WordPress penetration testing, Kali Linux is the standard tool for penetration testers.
+
```

Comment:
Add at least two more paragraphs here. This statement is too short for a major heading in an article.